hoomd/hpmc/UpdaterMuVT.h

```diff
@@ -1945,6 +1971,18 @@ bool UpdaterMuVT<Shape>::tryInsertParticle(uint64_t timestep,
         ShortReal r_cut_patch(0.0);
         Scalar r_cut_self(0.0);
 
+        if (field)
+            {
+            const BoxDim box = this->m_pdata->getGlobalBox();
+            lnboltzmann -= field->energy(box,
+                                         type,
+                                         pos,
+                                         quat<float>(orientation),
+                                         1.0, // diameter i
+                                         0.0 // charge i
```

Comment:
I am in general not a fan of hard-coding diameter and charge like this, however, all the rest of the code seems to follow this pattern so I guess we can leave it as is for now.

Reply:
Neither am I. However, for now, I think this should be updated when this ends up getting rewritten.